### PR TITLE
chore(.github/issue_template): add issue_template

### PR DESCRIPTION
**Pull request by wklken**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: "Feature Request"
+description: Suggest an enhancement to blueking-apigateway.
+title: "feat: ......"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        _提供足够详细的信息, 以便我们能够快速帮助你__
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]: 建议中文里不要混用英文标点，其他部分也有这个问题。

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+name: "Bug Report"
+description: Report a bug to help improve blueking-apigateway.
+title: "bug: ......"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        _提供足够详细的信息, 以便我们能够快速帮助你__
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:8]: 加个“请”？请提供足够……

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    id: version-environment
+    attributes:
+      label: Version and Environment
+      description: 版本及环境信息(Share your version and environment details)
+      value: |
+        - 蓝鲸版本号(blueking version):
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]: blueking version -> The version of BlueKing platform?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        - 容器化部署还是二进制部署(containerized or binary):
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:55]: containerized or binary -> Deployed on Kubernetes or using Binary Files

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        - 二进制部署请提供网关版本号(binary version of bk-apigateway):
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:56]: binary version of bk-apigateway -> The version of bk-apigateway, required for binary distribution

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        - 容器化部署请提供(Provide the following information if you are using containerized deployment):
+          - bk-apigateway helm chart version:
+          - value.yaml中是否做了修改(whether you have modified the value.yaml):
+          - value.yaml中的配置, 注意去除敏感信息(share the value.yaml, remove sensitive information):
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:60]: share the value.yaml, remove sensitive information -> 

The contents of the `values.yaml` file, remember to remove any sensitive information.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    attributes:
+      label: Version and Environment
+      description: 版本及环境信息(Share your version and environment details)
+      value: |
+        - 蓝鲸版本号(blueking version):
+        - 容器化部署还是二进制部署(containerized or binary):
+        - 二进制部署请提供网关版本号(binary version of bk-apigateway):
+        - 容器化部署请提供(Provide the following information if you are using containerized deployment):
+          - bk-apigateway helm chart version:
+          - value.yaml中是否做了修改(whether you have modified the value.yaml):
```
Comment (piglei) [inline .github/ISSUE_TEMPLATE/bug_report.yml:59]: 应该是 values.yaml？

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+### Description
+
+<!-- 关联相关issue Please include a summary of the change and which issue is fixed. -->
+<!-- 给出必要的上下文以及review需要的必要信息 Please also include relevant motivation and context. -->
+
+Fixes # (issue)
+
+### Checklist
+
+- [ ] 填写 PR 描述及相关issue (write PR description and related issue)
```
Comment (piglei) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: issue 前应空格，保持一致

Comment (wklken) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: done

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (wklken) [reply .github/ISSUE_TEMPLATE/bug_report.yml:59]: done

**Comment by piglei** [comment]
LGTM

**Review by piglei — APPROVED** [review]

**Review by zhu327 — APPROVED** [review]

**Review by alex-smile — APPROVED** [review]

**Review by alex-smile — APPROVED** [review]

**Review by alex-smile — APPROVED** [review]

**Pull request merged**